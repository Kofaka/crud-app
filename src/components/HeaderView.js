import React from 'react';

import logo from './logo.svg';
import './HeaderView.css';

const HeaderView = () => (
  <header className="header">
    <img src={logo} className="header-logo" alt="logo"/>
    <p>
      Welcome to React CRUD App
    </p>
  </header>
);

export default HeaderView;