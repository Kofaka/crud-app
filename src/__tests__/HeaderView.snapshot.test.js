import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme'

import HeaderView from '../components/HeaderView';

const setup = (propsOverrides) => {
  const props = Object.assign({}, propsOverrides);

  return {
    props
  };
};

describe('Component: HeaderView', () => {
    it('renders without crashing', () => {
        const {props} = setup();
        const view = shallow(<HeaderView {...props} />);
        expect(view).toMatchSnapshot();
    });
});

