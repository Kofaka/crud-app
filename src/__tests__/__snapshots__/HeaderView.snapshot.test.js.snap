// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: HeaderView renders without crashing 1`] = `
<header
  className="header"
>
  <img
    alt="logo"
    className="header-logo"
    src="test-file-stub"
  />
  <p>
    Welcome to React CRUD App
  </p>
</header>
`;
